GitUI/BranchTreePanel/RepoObjectsTree.Nodes.cs

```diff
@@ -124,12 +124,8 @@ private abstract class Tree : IDisposable
             private readonly CancellationTokenSequence _reloadCancellationTokenSequence = new();
             private bool _firstReloadNodesSinceModuleChanged = true;
 
-            // A flag to indicate whether the data is currently being filtered or not.
-            // This helps to reduce unnecessary treeview rebinds.
-            private bool _isCurrentlyFiltering;
-
             // A flag to indicate whether the data is being filtered (e.g. Show Current Branch Only).
-            private protected static AsyncLocal<bool> IsFiltering = new();
+            private protected AsyncLocal<bool> IsFiltering = new();
```

Comment:
Why did you remove the `static` modifier? Do we expect more than one instance of the left panel now?

Reply:
This replaces the `_isCurrentlyFiltering` flag, it would have been cumbersome to keep two flags for the same info.